api_docs/changelog.md

```diff
@@ -20,6 +20,12 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 240**
+
+* [`POST /message_edit_typing`](/api/set-message-edit-typing-status):
+  Added support for sending typing notification when a message is
+  being edited both in streams and direct messages.
```

Comment:
This should also document the new event type in `GET /events`.

Reply:
I updated it.